SteamController/SteamControllerManager.h

```diff
@@ -8,6 +8,9 @@
 
 #import <Foundation/Foundation.h>
 
+#define STEAMCONTROLLER_NO_SWIZZLING
+#define STEAMCONTROLLER_NO_PRIVATE_API
+
```

Comment:
```suggestion
```

don't enable these options by default, it should be possible to enable/disable them in build settings so it works with the `no-private-api` pod subspec